src/dataService/tagDataService.ts

```diff
@@ -140,35 +132,21 @@ export class TagDataService {
       .whereIn('tag', latestTags)
       .orderBy('_updatedAt', 'desc');
 
-    return tags.map(TagObjectMapper.mapDbModelToDomainEntity);
+    return tags.map(TagModel.toGraphqlEntity);
   }
 
   public async getTagsByName(names: string[]): Promise<Tag[]> {
```

Comment:
just checking if this method should call sanitizeTag as the old method does clean&validateTag

Reply:
no, since this is coming from the database any tags should already be sanitized

Reply:
just trying to understand our workflow here:

the `validateTag` function is being used by read calls as we call `toGraphqlEntity`

so in read, we only check if the required fields are set and we allow any invalid input to passthrough (e.g empty tag name as they are valid in our db as of now)

 whereas in write, we validate the tags format (e.g empty string/character limit etc) and block any invalid input (e.g empty tag string)

Reply:
Yep! you got it. The `toGraphQlEntity` validation I'm seeing as more of a developer check to make sure that we have included all the required fields in the data layer response, since we don't have type safety and also the db model does not match our graphql entity